Sentimental analysis preliminary stage

VinothkumarK24 opened the pull request:
In Sentimental Analysis script, We have 3 decisions 0 Neutral,+1 Good and -1 Bad, This will be updated to two decisions (Good and Bad), Once we finalize good and bad words, We should update the code.

Commit:
Sentimental analysis preliminary stage

laforgiamatt commented on Final Project/SentimentAnalysis.py:
```diff
@@ -0,0 +1,60 @@
+
+
+if __name__ == "__main__":
+    carddata,decisions= run('cardblend.txt')
```
Error here. Nothing creates cardblend

VinothkumarK24 replied:
Run updated scraper.py, Then run sentimental analysis file

VinothkumarK24 committed:
Update SentimentAnalysis.py

Please check now, First run updated  scraper.py then with Trainingcard.txt, run sentiment analysis.py.

VinothkumarK24 committed:
Update SentimentAnalysis.py

VinothkumarK24 reviewed (commented):
Please check now
